### DISCUSSION TITLE
Dúvida - Setar valores (p-value) na po-dynamic-form

### DISCUSSION THREAD

**Discussion by LeonardoDalcorso** [po-ui/po-angular#1523] [Angular]
**Boa tarde a todos !**
Estou montando um formulário dinâmico usando **(po-dynamic-form)** so que não estou conseguindo setar valores nos campos do formulario,
Por exemplo: Lista pre carregada, ao clicar para incluir é gerado um formulário, meu problema ocorre quando vou fazer um UPDATE, os valores não são setados nos campos carregados para o usuário fazer alteração.
Segui todo exemplo do Site, mas mesmo assim não esta funcionando.
Alguma dica?
Consigo passar o valor do campo diretamente no JSON?
Existe alguma outra forma de atualizar o p-value ?
**Muitos obrigado**

**Comment by deboraconstantino** [comment]
Vc pode fazer uma função pra validar a operação do componente e, quando for alteração, após receber todos os valores, atribuí-los para a propriedade do componente que está sendo passada para o p-values. Outra forma seria utilizar a propriedade p-load (Função ou serviço que será executado na inicialização do componente) para realizar essa carga.
